nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/CoreSession.java

```diff
@@ -477,6 +477,22 @@
      */
     DocumentModel createDocumentModel(String parentPath, String name, String typeName);
 
+    /**
+     * Creates a new document model using required information.
+     * <p>
+     * Used to fetch initial datamodels from the type definition.
+     * <p>
+     * DocumentModel creation notifies a {@link DocumentEventTypes#EMPTY_DOCUMENTMODEL_CREATED} so that core event
+     * listener can initialize its content with computed properties.
+     *
+     * @param parentRef the parent document ref {@link DocumentRef}
+     * @param name The destination name
+     * @param typeName the type name
+     * @return the initial document model
+     * @since 10.10-HF23
```

Comment:
```suggestion
     * @since 11.1
```

Reply:
I was thinking we should keep the same `since`, I mean by that if we did the modification on master -> `@since 11.1` we should keep it when we backport and I was thing this is also true from `10.10` to `11.1`, when the method is added on `10.10` and forwarded we should keep it.

Reply:
Not in this direction, what is important to know is the introduction in Master. Then you can change the since for backport if you want to be explicit, but it adds more work.